frontend/src/mixins/socket.ts

```diff
@@ -131,10 +131,15 @@ export default defineComponent({
     methods: {
 
         endpointDisplayFunction(endpoint : string) {
-            if (endpoint) {
-                return endpoint;
-            } else {
-                return this.$t("currentEndpoint");
+            for (const [k, v] of Object.entries(this.$data.agentList)) {
+                if (endpoint) {
+                    if (endpoint === v['endpoint'] && v['friendlyname'] !== "") {
+                        return v['friendlyname'];
+                    }
+                    if (endpoint === v['endpoint'] &&  v['friendlyname'] === "" ) {
+                        return endpoint;
+                    }
```

Comment:
Thank you so much for this pull request. Just a small suggestion to avoid [CI test failure](https://github.com/louislam/dockge/actions/runs/7748302986/job/21130673284?pr=414#step:10:94)
- Replaces single quotes with double quotes
- Remove duplicate space in Line 139
```suggestion
                    if (endpoint === v["endpoint"] && v["friendlyname"] !== "") {
                        return v["friendlyname"];
                    }
                    if (endpoint === v["endpoint"] && v["friendlyname"] === "") {
                        return endpoint;
                    }
```

Reply:
Oh sorry! Did not see your suggestions I already rebased. But thanks in advance!